index.bs

```diff
@@ -116,38 +153,65 @@ While a recent version of Edge identifies itself as:
-
-This document proposes a mechanism which might allow user agents to be a bit more aggressive about
-removing entropy from the `User-Agent` string generally by giving servers that really need some
-specific details about the client the ability to opt-into receiving them. It introduces four new
-Client Hints ([[I-D.ietf-httpbis-client-hints]]) that can provide the client's branding and version
-information, the underlying operating system's branding and major version, as well as details about
-the underlying device. Rather than broadcasting this data to everyone, all the time, user agents can
-make reasonable decisions about how to respond to given sites' requests for more granular data,
-reducing the passive fingerprinting surface area exposed to the network.
+    
+5.  Difference models of device perform very differently [[AlexRussell]]. Developers wishing to 
+    improve performance require this insight in analytics tools and in real time to optimise their
+    web site. The ability to optimise on first request is particularly important to authors targeting
+    people living in emerging economies where lower end device models and features are more widely
+    used than affluent western democracies.
+    
+6.  Vendors of devices and browsers use the information transmitted in `User-Agent` field values to 
+    measure the active install base of their products. This information in aggregate informs product
+    development decisions.
+    
+7.  Web analytics solutions and search engine tools such as those developed by Google 
+    [[Google-Analytics]] require the `User-Agent` field value to be present to provide insights into 
+    device classifications, models, browser and operating system versions.
+    
+8.  Identifying legitimate non human web traffic for the purposes of improving web sites based on 
+    human interactions only, and other purposes where including non human activity in analysis is 
+    undesirable, but where reliable identification of humans via sign in methods is either not
+    possible or desirable.
+
+9.  Fraud detection uses the version numbers, platform details, model information, and layout engine
+    versions, among others and form the basis for fingerprinting schemes for the identification and 
+    prevention of fraud or to track people.
+
+This document proposes a mechanism which will address the following distinct problems:
+
+1.  Address historic structural issues with the `User-Agent` field value by introducing new HTTP 
+    header fields and values.
+
+2.  Reduce the amount of information sent on every request.
+
+The document introduces seven new Client Hints ([[I-D.draft-ietf-httpbis-client-hints]]) that can 
+provide the client's branding and version information, the underlying operating system's branding 
+and major version, as well as details about the underlying device.
+
+Prior to deployment this approach will be compared with adopting a simpler and more efficient
```

Comment:
I'm not sure where this sentence comes from, or what it implies. Can you clarify please?